packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js

```diff
@@ -7,14 +7,15 @@
 
 const path = require( 'path' );
 const { tools, logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const parseGithubUrl = require( 'parse-github-url' );
 const cli = require( '../utils/cli' );
+const createGithubRelease = require( '../utils/creategithubrelease' );
 const displaySkippedPackages = require( '../utils/displayskippedpackages' );
 const executeOnPackages = require( '../utils/executeonpackages' );
 const generateChangelogForSinglePackage = require( './generatechangelogforsinglepackage' );
 const getPackageJson = require( '../utils/getpackagejson' );
 const getPackagesToRelease = require( '../utils/getpackagestorelease' );
 const getSubRepositoriesPaths = require( '../utils/getsubrepositoriespaths' );
-const releaseRepository = require( '../utils/releaserepository' );
```

Comment:
If the new code in this file does all the releasing why haven't you removed the `releaseRepository` file?

Reply:
We still have a task for releasing the single repository which uses this util.